Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py

```diff
@@ -2575,6 +2585,67 @@ def test_get_original_alerts_command(requests_mock):
     assert event.get('raw_log', {}).get('userIdentity', {}).get('accountId') == 'ID'  # assert vendor filter is correct
 
 
+@freeze_time("2022-05-03 11:00:00 GMT")
+def test_get_alert_by_filter(requests_mock, mocker):
```

Comment:
add a wrapper class for your tests to encapsulate them together:
```python
class TestGetAlertByFilter:
    @staticmethod
    def test_sanity():
        ...
```

Reply:
Done